pkg/ytdl/options_video.go

```diff
@@ -0,0 +1,57 @@
+package ytdl
+
+import (
+	"fmt"
+
+	"github.com/mxpv/podsync/pkg/config"
+	"github.com/mxpv/podsync/pkg/model"
+)
+
+type OptionsVideo struct {
+	quality   model.Quality
+	maxHeight int
+}
+
+func NewOptionsVideo(feedConfig *config.Feed) *OptionsVideo {
+	options := &OptionsVideo{}
+
+	options.quality = model.QualityHigh
+
+	if feedConfig.Quality == model.QualityLow {
+		options.quality = model.QualityLow
+	}
+
+	if feedConfig.MaxHeight > 0 {
+		options.maxHeight = feedConfig.MaxHeight
+	}
```

Comment:
Why do we need this if checks? why not just:
```go
options.quality = feedConfig.Quality
options.maxHeight = feedConfig.MaxHeight
```

(same for audio)

Reply:
I wanted to do more validations, but I didn't finish them. You're right, it needs to be cleaned up and brought back to a complete state.